projects/RabbitMQ.Client/client/impl/AutorecoveringConnection.cs

```diff
@@ -896,6 +896,12 @@ private void EnsureIsOpen()
         private void HandleTopologyRecoveryException(TopologyRecoveryException e)
         {
             ESLog.Error("Topology recovery exception", e);
+            if (e.InnerException is AlreadyClosedException || e.InnerException is OperationInterruptedException || e.InnerException is TimeoutException)
+            {
+                throw e;
+            }
+            ESLog.Error($"Will not throw Topology Exception {e.InnerException?.GetType().FullName}.", e);
```

Comment:
I assume this is mean to say "Will now throw". I also suggest a slightly different wording, something like

```
Will throw a topology recovery exception of type … to retry recovery
```

and perhaps it should be a warning and not an error?

Reply:
fixed in the last commit, my intention was to log why we have ignored the exception. And have something to search for if we want to add more exceptions in the future, I can also remove the logrow.

Reply:
let me know if i should make a pr to master also or instead,its exactly the same solution